docs/install_linux.md

```diff
@@ -117,6 +117,16 @@ Nix/NixOS users can install from [nixpkgs](https://search.nixos.org/packages?sho
 nix-env -iA nixos.gitAndTools.gh
 ```
 
+### Snaps
+
+Many Linux distro users can install using Snapd from the [Snap Store](https://snapcraft.io/gh) or the associated [repo](https://github.com/casperdcl/cli/tree/snap)
+
+```bash
+sudo snap install --edge gh && snap connect gh:ssh-keys
```

Comment:
can confirm this is the correct installation instruction

Reply:
The 6-hour updates are a generic snap mechanism and not related to just this specific Snap, right? That is why I didn't add that information. My logic was that I don't think that needs to be added, if that is the case, as it assumes an understanding of Snap basics. 

However, if you want, I can add it. Here is where I thought about adding it???

> _**Snaps are auto-updated every 6 hours.**_ `Snapd` is required and is available on a wide range of Linux distros.
>
> Find out which distros have it pre-installed and how to install it in the [Snapcraft Installation Docs](https://snapcraft.io/docs/installing-snapd)

Reply:
`snap connect gh:ssh-keys` is needed for authentication purposes. Is it needed for anything else besides that?  "Grand SSH access" seems too generic

I am thinking of added something like
```
Note: `snap connect gh:ssh-keys` is needed for all authentication and SSH needs
```